src/test/kotlin/fr/social/gouv/agora/usecase/qag/GetQagModeratingListUseCaseTest.kt

```diff
@@ -244,12 +311,104 @@ internal class GetQagModeratingListUseCaseTest {
                     "qagId6" to QagStatus.OPEN,
                 ),
                 responseQagIdList = listOf("qagId1"),
+                qagLockedList = emptyList(),
+                expectedModeratingQagCount = 2,
+            ),
+            input(
+                qagIdAndStatusList = listOf(
+                    "qagId1" to QagStatus.OPEN,
+                    "qagId2" to QagStatus.ARCHIVED,
+                    "qagId3" to QagStatus.MODERATED_ACCEPTED,
+                    "qagId4" to QagStatus.MODERATED_REJECTED,
+                    "qagId5" to QagStatus.OPEN,
+                    "qagId6" to QagStatus.OPEN,
+                ),
+                responseQagIdList = listOf("qagId1"),
+                qagLockedList = listOf(Pair("qagId1", "otherUser")),
```

Comment:
pour les Pair, tu peux les instancier comme pour `qagIdAndStatusList ` :
`listOf("qagId1" to otherUserId)`

Reply:
Et c'est dommage de ne pas utiliser la variable que tu as créé, tu peux la déplacer dans le `companion object` si besoin

Reply:
c'est fait